### DISCUSSION TITLE
Chartbrew v2 Feedback & Discussion

### DISCUSSION THREAD

**Discussion by razvanilin** [chartbrew/chartbrew#138] [General]
Hey everyone,

Since the v2 has just been released, I want to gather a list of feedback to improve the platform moving forward. I hope you enjoy the new UI!

You can also leave questions that I will either respond to here or during the live stream on Tuesday. You can [check the details here](https://www.eventbrite.com/e/418595720067).

Looking forward to hearing what you think of the new version! ✌️

**Comment by DrZoidberg09** [comment]
v2 looks amazing! Great progress!

Two things that come to my mind:

- Again OAuth2 / OpenID Connect features. That would make integrating Chartbrew in exisitng environments super handy.
- Static text / Markdown fields for the dashboard. This would allow commenting / explaining the charts to 3rd party users  much easier

Probably more things will come to my mind :)
